src/test/java/org/opensearch/security/SlowIntegrationTests.java

```diff
@@ -30,6 +30,7 @@
 
 package org.opensearch.security;
 
+import org.apache.http.Header;
```

Comment:
Why are there new imports but not being used?

Reply:
I have a second PR coming soon with changes to admin:admin password.  I reverted changes I had made to unit tests, so Header will be used in the next PR.  I just missed it on this one.

Reply:
In the meantime, I'll remove the ones that aren't currently being used